go.mod

```diff
@@ -1,4 +1,4 @@
-module sigs.k8s.io/knftables
+module github.com/aroradaman/knftables
```

Comment:
you shouldn't need to do this... `make test` should work anyway... what problem where you having?

Reply:
I was planning to point to my fork in k/k and then run `pull-kubernetes-e2e-gci-gce-nftables` for testing

Reply:
will drop the dummy commit after testing.

Reply:
ok, you probably just need to use the `replace` section of `go.mod` in the k/k PR. (The existing replacements are all just for the staging repos, but you can add a line like:

    sigs.k8s.io/knftables => github.com/aroradaman/knftables TAG-OR-HASH

(and then run `go mod tidy` if you specified a hash, to make it convert it to a fake tag)

Reply:
This + `./hack/update-vendor.sh` did the trick.